pysd/py_backend/vensim/vensim2py.py

```diff
@@ -49,17 +49,21 @@ def get_file_sections(file_str):
     Examples
     --------
     >>> get_file_sections(r'a~b~c| d~e~f| g~h~i|')
-    [{'returns': [], 'params': [], 'name': 'main', 'string': 'a~b~c| d~e~f| g~h~i|'}]
+    [{'returns': [], 'params': [], 'name': 'main', 'string': 'a~b~c| d~e~f|
```

Comment:
does doctests handle this linebreak?

Reply:
No, it is not. I am recovering the previous version and improving some functions docs.